src/calendars/portugal.js

```diff
@@ -127,7 +127,7 @@ let dates = year => {
       }
     },
     {
-      "key": "blessedSanchaAndMafaldaVirginsOrBlessedTheresaOfPOrtugalReligious",
+      "key": "blessedSanchaAndMafaldaVirginsOrBlessedTheresaOfPrtugalReligious",
```

Comment:
Could there be another typo here?

`blessedSanchaAndMafaldaVirginsOrBlessedTheresaOfPrtugalReligious`
might have intended to be
`blessedSanchaAndMafaldaVirginsOrBlessedTheresaOfPortugalReligious`
instead

Reply:
You’re right. I made a mistake there. :smiley: